src/client/components/Reactions/ReactionsList.js

```diff
@@ -27,6 +27,10 @@ export default class UserList extends React.Component {
     const { votes, ratio } = this.props;
     const defaultPageItems = 20;
     const noOfItemsToShow = defaultPageItems * this.state.page;
+    const voteValue = vote =>
+      isNaN((vote.rshares_weight || vote.rshares) * ratio)
+        ? 0
+        : (vote.rshares_weight || vote.rshares) * ratio;
```

Comment:
it can looks like this:
    const voteValue = vote => ((vote.rshares_weight || vote.rshares) * ratio) || 0;